src/transaction_signature_options/sign_with_ledger/mod.rs

```diff
@@ -35,7 +35,7 @@ pub struct SignLedger {
 #[derive(Clone)]
 pub struct SignLedgerContext {
     network_config: crate::config::NetworkConfig,
-    signed_transaction: near_primitives::transaction::SignedTransaction,
+    signed_transaction: super::SignedTransactionOrSignedDelegateAction,
```

Comment:
Rename the field to `signed_transaction_or_signed_delegate_action` (apply the same to all the other similar places)